library/common/include/c_types.h

```diff
@@ -130,10 +162,11 @@ typedef void (*on_error)(envoy_error error, void* context);
  * Interface that can handle HTTP callbacks.
  */
 typedef struct {
-  on_headers on_headers_f;
-  on_data on_data_f;
-  on_metadata on_metadata_f;
-  on_trailers on_trailers_f;
-  on_error on_error_f;
+  envoy_on_headers_f on_headers;
+  envoy_on_data_f on_data;
+  envoy_on_metadata_f on_metadata;
+  envoy_on_trailers_f on_trailers;
+  envoy_on_error_f on_error;
+  atomic_bool *canceled;
```

Comment:
I think the linter might complain about this

Reply:
about which?